samza-core/src/main/scala/org/apache/samza/storage/ContainerStorageManager.java

```diff
@@ -183,15 +183,17 @@ public ContainerStorageManager(
 
     this.storeDirectoryPaths = new HashSet<>();
 
-    this.inMemoryStores = ContainerStorageManagerUtil.createInMemoryStores(changelogSystemStreams, storageEngineFactories, sideInputStoreNames, storeDirectoryPaths, containerModel,
-        jobContext, containerContext, taskInstanceMetrics, taskInstanceCollectors, serdes, storageManagerUtil, loggedStoreBaseDirectory, nonLoggedStoreBaseDirectory, config
-    );
+    this.inMemoryStores = ContainerStorageManagerUtil.createInMemoryStores(
+        activeTaskChangelogSystemStreams, storageEngineFactories, sideInputStoreNames,
```

Comment:
Note: Changed changelogSystemStreams to activeTaskChangelogSystemStreams here for correctness.